README.md

```diff
@@ -35,12 +35,10 @@ if data, err := MyFunc(); err != nil {
 
     //....
 
-    if x, ok := err.(*xerr); ok {
-        // In this example we only interested in the last 5 execution calls within the stack
-        fmt.Println(x.Details(5)) // Details prints cause error, mask if specified, and stack (accepting the maximum stack height as parameter)
-    } else {
-        fmt.Println(err) // print basic error message if it is original error
-    }
+    // Details prints cause error, mask if specified, and stack (accepting the maximum stack height as parameter)
```

Comment:
// `Details` returns cause error, mask if specified, and the stack (accepting the maximum stack height as parameter)
// In this example only 5 calls in the stack will be printed out